mkdocs.yml

```diff
@@ -43,6 +44,9 @@ watch:
 
 nav:
   - "index.md"
+  - Pipeline:
+    - "exomiser_pipeline.md"
```

Comment:
Doesn't this have to be  key value pairs? How would mkdocs know otherwise how to render the link?

Reply:
The link will be based on the file's name and the nav title is the file header content.